src/components/ButtonsWithPhrases/ButtonsWithPhrases.jsx

```diff
@@ -0,0 +1,15 @@
+import './ButtonsWithPhrases.css';
+
+// eslint-disable-next-line react/prop-types
+const ButtonsWithPhrases = ({buttonName}) => {
```

Comment:
Not really proper name for the component in general it's button and every button can has a description, not smth WithPhrase